go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/canonical/pebble
 go 1.22
 
 require (
+	github.com/GehirnInc/crypt v0.0.0-20230320061759-8cc1b52080c5
```

Comment:
Is this package maintained? Are we confident about introducing it in every charm / pebble deployment out there?

Reply:
It hasn't been recently changed, but the part of it we're using is small a straight-forward, and the algorithm being implemented is stable and well-documented ([SHA-crypt](https://www.akkadia.org/drepper/SHA-crypt.txt)). So I would say it's "stable" rather than "unmaintained". I'm actually contemplating vendoring this (the core part of it is only ~100 LoC). But open to other ideas/concerns too.

Reply:
We've done some diligence on it, see https://github.com/canonical/pebble/pull/563#issuecomment-2635646862 for example.